CheckDrive.Mobile/CheckDrive.Mobile/CheckDrive.Mobile/Stores/Account/MockAccountStore.cs

```diff
@@ -6,11 +6,33 @@ namespace CheckDrive.Mobile.Stores.Account
 {
     internal class MockAccountStore : IAccountStore
     {
+        private AccountDto _account;
+
+        public MockAccountStore()
+        {
+            _account = new AccountDto
+            {
+                FirstName = "John",
+                LastName = "Doe",
+                Email = "john.doe@example.com",
+                Passport = "1234567891987654",
+                PhoneNumber = "+123456789",
+                Login = "johndoe",
+                Address = "123 Main St, Cityville",
+                Birthdate = new System.DateTime(1990, 1, 1),
+            };
```

Comment:
Why don't you use Bogus library? We already have a helper class **FakeDataGenerator** which generates some fake data. You add some additional methods there if you need, or you can reuse what's already there.